lib/internal/http2/core.js

```diff
@@ -289,11 +289,9 @@ function onSessionRead(nread, buf, handle) {
   _unrefActive(this); // Reset the session timeout timer
```

Comment:
This commit (78be06c9d114e4a297dd8c21de2de3154445e3ce) has an accidental file mode change ;)